### PR TITLE
Exercise-2: Tests

### PR DESCRIPTION
**Pull request by pasha-dektev**
1. Android tests
2. Unit tests

### CODE REVIEW

File: app/src/androidTest/java/com/lightricks/feedexercise/data/FeedRepositoryTest.kt [hunk 1]
```diff
@@ -1,13 +1,131 @@
...
+    }
+
+    @After
+    fun tearDown() {
+        db.close()
+        Dispatchers.resetMain()
+    }
+
+    @Test
+    fun repo_refreshFeed_success() = runTest(dispatcher) {
```
Comment (nissimLightricks) [inline app/src/androidTest/java/com/lightricks/feedexercise/data/FeedRepositoryTest.kt:71]: In general we are using the following naming convention for tests MethodName_StateUnderTest_ExpectedBehavior

Comment (nissimLightricks) [reply app/src/androidTest/java/com/lightricks/feedexercise/data/FeedRepositoryTest.kt:71]: https://dzone.com/articles/7-popular-unit-test-naming

**Review by nissimLightricks — APPROVED** [review]
LGTM

File: app/src/androidTest/java/com/lightricks/feedexercise/data/FeedRepositoryTest.kt (hunk 1 above)
Comment (pasha-dektev) [reply app/src/androidTest/java/com/lightricks/feedexercise/data/FeedRepositoryTest.kt:71]: noted

**Pull request merged**